openedx/core/djangoapps/credit/api.py

```diff
@@ -322,6 +320,40 @@ def get_credit_requests_for_user(username):
     return CreditRequest.credit_requests_for_user(username)
 
 
+def get_credit_requirement(course_key, namespace, name):
+    """Returns the requirement of a given course, namespace and name.
+
+    Args:
+        course_key(CourseKey): The identifier for course
+        namespace(str): Namespace of requirement
+        name: Name of the requirement
+
+    Returns:
+        'CreditRequirement' object
+
+    """
+    return CreditRequirement.get_course_requirement(course_key, namespace, name)
+
+
+def set_credit_requirement_status(username, requirement, status="satisfied", reason=None):
+    """Update Credit Requirement Status for given username and requirement
+        if exists else add new.
+
+    Args:
+        username(str): Username of the user
+        requirement(CreditRequirement): 'CreditRequirement' object
+        status(str): Status of the requirement
+        reason(dict): Reason of the status
+
+    Returns:
+        'CreditRequirementStatus' object
+
+    """
+    return CreditRequirementStatus.add_or_update_requirement_status(
```

Comment:
You can address @wedaly's comment ("We shouldn't need to return `created` to other Django apps.") as follows:
```python
requirement_status, _ = CreditRequirementStatus.add_or_update_requirement_status(
    username, requirement, status, reason
)

return requirement_status
```